helm/values.yaml

```diff
@@ -90,28 +90,32 @@ ingress:
     #    hosts:
     #      - chart-example.local
   core:
-    enabled: false
+    enabled: true
     className: ""
-    annotations: {}
-      # kubernetes.io/ingress.class: nginx
-      # kubernetes.io/tls-acme: "true"
+    annotations:
+      kubernetes.io/ingress.class: traefik-internal
+      traefik.ingress.kubernetes.io/router.entrypoints: web,websecure
+      cert-manager.io/cluster-issuer: letsencrypt-dns01
+      external-dns.alpha.kubernetes.io/target: traefik-internal.parity-stg.parity.io.
+      traefik.ingress.kubernetes.io/router.tls: "true"
     hosts:
-      - host: chart-example.local
+      - host: feed.telemetry.parity-stg.parity.io
         paths:
           - path: /
             pathType: ImplementationSpecific
-    tls: []
-    #  - secretName: chart-example-tls
-    #    hosts:
-    #      - chart-example.local
+    tls:
+      - secretName: feed.telemetry.parity-stg.parity.io
+        hosts:
+          - feed.telemetry.parity-stg.parity.io
```

Comment:
Aah, I guess this is what allows WSS to work?

Reply:
yep! this is the configuration for the ingress controller, it automatically proxies the connections to this domain to the core service with a SSL certificate.